src/modules/commons/modelActions.ts

```diff
@@ -0,0 +1,27 @@
+import * as api from '../../api'
+import * as type from './types'
+
+const modelActions = {
+	getModel: (modelName: string, id?: number) => dispatch => {
+		dispatch({
+			type: type.get.start(modelName),
+		})
+
+		api[modelName].get(id).then(
```

Comment:
It's ok but it is not generic as people dont know they have to name their model exactly the same as the name they put in the api